main.tf

```diff
@@ -143,9 +144,7 @@ module "stacks" {
 
 # `administrative` policies are always attached to the `administrative` stack
 # `spacelift_current_stack` is the administrative stack that manages all other infrastructure stacks
-data "spacelift_current_stack" "administrative" {
-  count = var.external_execution ? 0 : 1
```

Comment:
Might as well remove this input var too

Reply:
yeah that's a good point @aknysh , this will fail for first time executions outside of Spacelift. I'll have to keep this in and adjust the logic a bit to account for it, and I should be able to handle that today. Good catch.